src/cogserver/vrt.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Literal, Optional
+from enum import Enum
+from fastapi import APIRouter, Query, Response
+from titiler.core.factory import FactoryExtension, TilerFactory, BaseTilerFactory, MultiBaseTilerFactory
+from osgeo import gdal
+import tempfile
+
+router = APIRouter()
+
+
+class ResampleAlg(Enum):
+    nearest: str = "nearest"
+    bilinear: str = "bilinear"
+    cubic: str = "cubic"
+    cubicspline: str = "cubicspline"
+    lanczos: str = "lanczos"
+    average: str = "average"
+    mode: str = "mode"
+
+
+async def create_vrt_from_urls(
+        urls: List[str],
+        xRes: float = 0.1,
```

Comment:
`create_vrt_from_urls` can also be moved to `extensions/vrt.py` folder since it is only used from `VRTExtension` class.